.github/scripts/install-rocksdb.sh

```diff
@@ -9,7 +9,7 @@ git clone https://github.com/facebook/rocksdb.git /home/runner/rocksdb
 
 cd /home/runner/rocksdb
 
-git checkout v7.7.3
+git checkout v7.9.2
```

Comment:
this should be pulled from the environment and set to v7.10.2 (another benefit to rebasing off of https://github.com/Kava-Labs/kava/pull/1532/files)